charts/lagoon-logs-concentrator/templates/secret.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     {{- include "lagoon-logs-concentrator.labels" . | nindent 4 }}
 stringData:
   FORWARD_SHARED_KEY: {{ required "A valid .Values.forwardSharedKey required!" .Values.forwardSharedKey }}
+  LOGSDB_ADMIN_USER: {{ default "admin" .Values.elasticsearchAdminUser }}
```

Comment:
Minor style thing, but could we change this to have the default value in the `values.yaml`, and just reference `{{ .Values.elasticsearchAdminUser }}` here? This is the style we have used elsewhere in the chart.